js/script.js

```diff
@@ -1 +1,38 @@
-console.log('Hello, front end');
+
+document.addEventListener('DOMContentLoaded', function() {
+  console.log("DOM loaded");
+
+    var form = document.getElementById('conversionForm');
+    var input = document.getElementById('quantity');
+    var convertButton  = document.getElementById('convertButton');
+    var resetButton = document.getElementById('resetButton');
+    var fButton = document.getElementById('fButton');
+    var cButton = document.getElementById('cButton');
+    var tempDisplay = document.getElementById('tempDisplay');
+
+    resetButton.addEventListener('click', function(){
+      tempDisplay.innerHTML=" ";
+    })
+
+    resetButton.addEventListener('click', function(){
+      quantity.value=" ";
+    })
+
+
+
+
+
+    form.addEventListener('submit', function(x){
+      x.preventDefault();
+
+      // console.log(input.value);
+    tempDisplay.innerHTML = c(input.value);
+    });
+
+});
+
+function c(f) {
```

Comment:
Good idea putting the conversion in a separate function, more modular. You could even put the function itself inside your "domContentLoaded" event function if you didn't want to pollute the global scope. Also watch out for naming, "convertToCelcius" is easier to make sense of than "c"